wear/src/main/java/com/alexstyl/specialdates/MainActivity.java

```diff
@@ -0,0 +1,18 @@
+package com.alexstyl.specialdates;
```

Comment:
Some heads up for follow up PRs. The project's packages as structured per feature. :)